client/my-sites/sharing/connections/connection.jsx

```diff
@@ -170,4 +186,13 @@ module.exports = React.createClass( {
 			</li>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state, { connection } ) => ( {
+		connectionUser: getUser( state, connection.keyring_connection_user_ID ),
```

Comment:
Has Redux user state been fleshed out enough that this will return users with similar consistency to before (`site.getUser`)?